.buildkite/macos.integration.pipeline.yml

```
@@ -0,0 +1,98 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+common:
+  - vault_ec_key_prod: &vault_ec_key_prod
+      elastic/vault-secrets#v0.1.0:
+        path: "kv/ci-shared/platform-ingest/platform-ingest-ec-prod"
+        field: "apiKey"
+        env_var: "EC_API_KEY"
+
+steps:
+  - label: "Packaging: darwin/arm64"
+    key: packaging-macos-arm64
+    env:
+      PLATFORMS: "darwin/arm64"
+      PACKAGES: "tar.gz"
+    command: ".buildkite/scripts/steps/integration-package.sh"
+    artifact_paths:
+      - build/distributions/**
+    retry:
+      automatic:
+        limit: 1
+    agents:
+      provider: "aws"
+      instanceType: "c6g.4xlarge"
+      imagePrefix: "core-ubuntu-2204-aarch64"
+
+  - label: Start ESS stack for macOS
+    key: integration-macos-ess
+    depends_on: packaging-macos-arm64
+    env:
+      ASDF_TERRAFORM_VERSION: 1.9.2
+    command: |
+      #!/usr/bin/env bash
+      set -euo pipefail
+      source .buildkite/scripts/steps/ess_start.sh
+    artifact_paths:
+      - test_infra/ess/*.tfstate
+      - test_infra/ess/*.lock.hcl
+    agents:
+      image: "docker.elastic.co/ci-agent-images/platform-ingest/buildkite-agent-beats-ci-with-hooks:0.5"
+      useCustomGlobalHooks: true
+    plugins:
+      - *vault_ec_key_prod
+
+  - group: "Stateful: MacOS"
+    key: integration-tests-macos
+    depends_on:
+      - integration-macos-ess    
```

Can you double-check if there are any cases with the right extra spaces? I've seen that in the past; maybe there's something in your IDE causing them?


```suggestion
      - integration-macos-ess
```